src/plone/restapi/testing.py

```diff
@@ -57,20 +56,6 @@
 else:
     PLONE_5 = True  # pragma: no cover
```

Comment:
I think we can also drop this, right?

Reply:
I think the ``PLONE_VERSION`` should also not be necessary, some lines above.

Reply:
@thet Good catch! Removed it.